### ISSUE TITLE
Merge xslt and add support for extensions

### ISSUE THREAD

**Issue by jeroen** [github/cmark#114]
The upstream repo now has a very cool [xml2md.xsl](https://github.com/commonmark/cmark/blob/master/tools/xml2md.xsl) file thanks to @nwellnhof that converts the xml ast back to markdown. This allows for editing a markdown document via the xml tree using e.g. xpath, and then convert it back to markdown.

It would be great if this xslt could be merged and updated to support the GFM extensions.

**Comment by kivikakk** [comment]
Agreed! I'll get the upstream XSLT when we next merge upstream, but updating it to work with GFM extensions is unfortunately beyond my capabilities. (As always, PRs gratefully accepted!)

**Comment by maelle** [comment]
I think this can be closed now 😺
